src/shared/syscheck_op.c

```diff
@@ -1517,7 +1517,8 @@ cJSON *win_perm_to_json(char *perms) {
         char *username = perm_node;
         perm_node = strchr(perm_node, '(');
         if (!perm_node) {
-            goto error;
+            mdebug2("Uncontrolled condition when parsing a Windows permission from '%s'. Skip permission", username);
```

Comment:
```suggestion
            mdebug1("Uncontrolled condition when parsing the username from '%s'. Skipping permission.", username);
```

It is better to use debug1 since it is an unexpected situation.  In addition, let's be more specific in the info we are parsing to differenciate the errors.